pages/index.tsx

```diff
@@ -8,9 +8,13 @@ import SummaryStats from "@/components/SummaryStats"
 import Footer from "@/components/Footer"
 import TickerSearch from "@/components/TickerSearch"
 import CompanyOverview from "@/components/CompanyOverview"
+import HistoricalByDay from "@/components/HistoricalByDay"
+import Tabs from "@/components/Tabs"
 
 const appTitle = "US Bank - Stock Analytics"
 
+// todo: hide tabs until a ticker search exists
```

Comment:
Handle this with react-rxjs <Subscribe />

Reply:
@gracescharf - apologies for sending you down the rabbit hole, but `Subscribe` is likely to not the approach you want.  We can discuss at 9:45